### ISSUE TITLE
no-internal-modules cant allow deep import

### ISSUE THREAD

**Issue by SergeyKatugin** [benmosher/eslint-plugin-import#1949]
Hi everyone.
I have a import with deep relative path. like this
`import { Service } from '../../../services/my-service/help.service';`

And i wanna allow any import with pattern `.service';`.

I try to make a glob for it, and it not working.

I made glob `**/**.service';`
And it works with not deep import like 
`import { Service } from '../services/my-service/help.service';`

May i wrong? or it's not supports

**Comment by ljharb** [comment]
Can you provide a simplified repro repo?

**Comment by SergeyKatugin** [comment]
@ljharb yes. https://github.com/SergeyKatugin/eslint-import-plugin-repro

file https://github.com/SergeyKatugin/eslint-import-plugin-repro/blob/main/src/helper/my-helper/helper.ts

**Comment by ljharb** [comment]
@SergeyKatugin that repo doesn't have an eslint config. How can i reproduce the issue?
